mailqueue/models.py

```diff
@@ -125,11 +125,13 @@ def _send(self):
 
             # Add any additional attachments
             for attachment in self.attachment_set.all():
-                path = attachment.file_attachment.path
-                if os.path.isfile(path):
-                    with open(path, 'rb') as f:
-                        content = f.read()
-                    msg.attach(attachment.original_filename, content, None)
+                # Try different ways to read the file
+                try:
+                    self._attach_local_storage_file(msg, attachment)
+                except NotImplementedError:
```

Comment:
I guess it would be preferable to check which storage is used based on django settings. On the other hand this needs to work and we might remove mail queue anyways...

Reply:
Yes, I've also tried to have logic per storage in the previous commit. But then decided to make it simple for now. There's a chance that we'll still use this package with file system storage. The fix here is compatible with both S3 and local filesystem. So it should be good to deploy.